addons/l10n_vn/data/template/account.tax.group-vn.csv

```diff
@@ -2,3 +2,4 @@
 "tax_group_0","VAT 0%","base.vn","Thuế GTGT 0%"
 "tax_group_5","VAT 5%","base.vn","Thuế GTGT 5%"
 "tax_group_10","VAT 10%","base.vn","Thuế GTGT 10%"
+"tax_group_exemption","0% VAT EXEMPT","base.vn","Không thuộc đối tượng chịu thuế GTGT"
```

Comment:
Let's wait for me to ask some accountant to verify this

Reply:
`0% VAT EXEMPT` is good. Translation is up to you though :slightly_smiling_face:

Reply:
Yes i'll keep that after consulting from my accountant -> Mark resovled

Reply:
@aliyatastemirova @Levizar after consulting both my accountant and some law, can i exclude the `0%` and leave `VAT EXEMPTION` only because in VietNam `0% VAT` vs `VAT EXEMPTION` is different, user really do not want the `0%` going to display on the invoice like this
![2024-04-16_09-18](https://github.com/odoo/odoo/assets/56789189/76c15b20-de30-44b8-a9be-08e7beb1e4d1)

Reply:
thanks for clarifying, what do you think about `VAT EXEMPT` then?

Reply:
@aliyatastemirova i think `VAT EXEMPTION` is way more correct in english right ?

Reply:
you were right, my bad.
Our 0% taxonomy is for specific stuff virtually ending in a 0% tax but that isn't exempted by itself.